### DISCUSSION TITLE
CIP 33 - Multi Currency Support - Discussions

### DISCUSSION THREAD

**Discussion by YazzyYaz** [celo-org/celo-proposals#337] [CIP Discussions]
Discussions for CIP 33: Multicurrency Support
Proposal [here](https://github.com/celo-org/celo-proposals/issues/125).

**Comment by zviadm** [comment]
Do you prefer comments here, or in the notion doc that was shared before: https://www.notion.so/Multi-Currency-Support-06fffb1ecc0b429fa3682ba9843d5d19 ? (I left two comments on the notion doc, but can copy paste them here too)

**Comment by sissnad** [comment]
I think it would be good to have all comments here. My feeling is the notion page will get much less traffic
